pages/partials/submission.ejs

```diff
@@ -24,6 +24,11 @@
   <div class="collapse <% if (submissionIdx == submissionCount) { %>show<% } %>" id="submission-<%= submission.id %>-body">
     <div class="card-body submission-body" >
       <%- submissionHtml %>
+      <% if (typeof submission != 'undefined' && typeof submission.pdf_barcode_scan != 'undefined') { %>
+        <div class="submission-body-pdf-barcode-scan-container">
+          <iframe class="submission-body-pdf-barcode-scan" width="100%" type="text/html" src="data:application/pdf;base64,<%= submission.pdf_barcode_scan %>"></iframe>
```

Comment:
Are there missing custom styles for these? I see classes but no corresponding CSS styles.

Reply:
Done. Taken out.

Reply:
Added one back in for a test that uses it.